Configs/Module-Shared.xcconfig

```diff
@@ -230,6 +230,7 @@ SKIP_INSTALL = YES
 
 // The list of supported platforms from which a base SDK can be used. This setting is used if the product can be built for multiple platforms using different SDKs.
 SUPPORTED_PLATFORMS = macosx iphoneos iphonesimulator watchsimulator watchos appletvsimulator appletvos
+ALLOW_TARGET_PLATFORM_SPECIALIZATION = YES
```

Comment:
Small Nitpick:
We sort the settings alphabetically in this file for easier lookup. Can you move this one to the beginning please?
Also, add a small comment above it, what exactly it does.

Reply:
Done, thanks!